### DISCUSSION TITLE
Cannot connect to Discord server

### DISCUSSION THREAD

**Discussion by sophokles73** [eclipse-zenoh/roadmap#60] [Q&A] [answered]
When following [the link to join the Discord server](https://zenoh.io/community/) on the project website, Discord tells me that I cannot join because the invite has expired or I do not have the required authority to join. Is the Discord server still supposed to be used to get in touch and ask a question or should I use GitHub Discussions instead?

**Comment by Mallets** [comment]
Hello @sophokles73 , 

I confirm that the Discord server is still very well used and supported.
While we are looking into the invitation link problem, feel free to use GitHub discussions!

**Comment by J-Loudet** [comment, marked as answer]
Hi @sophokles73,

While we update the relevant parts on the website, the following link should work: https://discord.gg/2GJ958VuHs
Do Keep us posted if you still have any issue!

**Reply by sophokles73** [reply]
Indeed, the new link works :-) Thanks a lot.
